### PR TITLE
Minor Integration test fixes

### PR DESCRIPTION
**Pull request by alextwoods**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

1. To make sure we include your contribution in the release notes, please make sure to add description entry for your changes in the "unreleased changes" section of the `CHANGELOG.md` file (at corresponding gem). For the description entry, please make sure it lives in one line and starts with `Feature` or `Issue` in the correct format.

2. For generated code changes, please checkout below instructions first:
  https://github.com/aws/aws-sdk-ruby/blob/version-3/CONTRIBUTING.md

Thank you for your contribution!

### CODE REVIEW

File: gems/aws-sdk-support/features/smoke_step_definitions.rb [hunk 1]
```diff
@@ -21,7 +21,8 @@
   begin
     @regional_client.send(operation.to_sym, opts)
     @operation_raised_error = false
-  rescue StandardError
+  rescue StandardError => e
+    puts "Got an error: #{e}"
```
Comment (mullermp) [inline gems/aws-sdk-support/features/smoke_step_definitions.rb]: Debugging

File: tasks/test.rake [hunk 2]
```diff
@@ -47,15 +47,18 @@ end
 
 rule /^test:features:.+$/ do |task|
   dir = "gems/#{task.name.split(':').last}/features"
-  tags = "-t 'not @veryslow'"
+  # Exclude support smoke tests as these require account settings
+  # the logical or with not results in skipping tests marked with
+  # @support AND @smoke only.
+  tags = "-t 'not @veryslow' -t 'not @support or not @smoke'"
```
Comment (mullermp) [inline tasks/test.rake:53]: Is there another way to meet this condition?

**Review by mullermp — APPROVED** [review]

File: tasks/test.rake (hunk 2 above)
Comment (alextwoods) [reply tasks/test.rake:53]: I'm not sure - this was the only way I could find to write this.

File: gems/aws-sdk-support/features/smoke_step_definitions.rb (hunk 1 above)
Comment (alextwoods) [reply gems/aws-sdk-support/features/smoke_step_definitions.rb]: Removed

File: tasks/test.rake (hunk 2 above)
Comment (alextwoods) [reply tasks/test.rake:53]: I did test that all other smoke tests are run.  And that other support tests would be run (we don't have any.  but if we wrote them in the future they would be run.  I'm not sure if thats what we want or not.  but I was trying to make a very selective filter).

**Pull request merged**